JobParseAndNormalize.md

```diff
@@ -0,0 +1,353 @@
+- [Versioning](#versioning)
+
+##Summary
+
+The Job Parse and Normalize (JPAN) service parses a Base64-encoded job posting and further enriches the parsed data with the following normalizations and classifications.  
+
+ #####Enrichments:
+ - Job Title Classifications
+    - [Carotene](https://github.com/cbdr/DataScienceAPIDocumentation/blob/master/JobTitle.md) <sup>(optional)</sup>
+    - [ONET](https://github.com/cbdr/DataScienceAPIDocumentation/blob/master/JobTitle.md) <sup>(optional)</sup>
+    - Textkernel
+ - [Normalized Company Name](https://github.com/cbdr/DataScienceAPIDocumentation/blob/master/CompanyNormalization.md)  <sup>(optional)</sup>
+ - Skills Extraction
+    - [Skills 4.0](https://github.com/cbdr/DataScienceAPIDocumentation/blob/master/Skills.md) <sup>(optional)</sup>
+    - Textkernel Skills
+ - Language Skills
+ - [Company Geography](https://github.com/cbdr/DataScienceAPIDocumentation/blob/master/Geography.md) <sup>(optional)</sup>
+ - [Job Geography](https://github.com/cbdr/DataScienceAPIDocumentation/blob/master/Geography.md) <sup>(optional)</sup>
+ - [Job Level](https://github.com/cbdr/DataScienceAPIDocumentation/blob/master/JobLevel.md) <sup>(optional)</sup>
+ - Education Level
+ - Employment Level
+ - Contract Type
+ 
+
+Enrichments marked as <sup>__(optional)__</sup> must be requested using the `desired_enrichments` parameter, as they require additional calls to other services. For example, a client who only needs Job Title and Skills classifications could structure a Job Parse and Normalize request that would only retrieve these classifications. This parameter is documented in further detail below.
+
+The service is located at https://api.careerbuilder.com/core/parsing/normalizedjob. As usual, you will need OAuth core credentials to use this service. (*If you do not have these, please go [here](http://apitester.cbplatform.link/credentials) or email PlatformSoftware@careerbuilder.com to request core credentials.*)
+
+##Request Structure
+
+This service supports the HTTP GET and POST methods. Because Base64-encoded documents can be quite large, POST is encouraged for production use.
+
+The following parameters may be supplied in the query string (for HTTP GET) or form body (for HTTP POST):
+
+* **document** (Required) -- A .doc, .docx, .pdf, .rtf, .txt, .odt, .wps, and .pages documents given in a BASE64 encoded string.  Please note that the .pages format is not accepted by Textkernel; you will need to specify another parser.
+
+* **desired_enrichments** (Required) -- A comma-separated list, without spaces, of the desired normalization calls to perform on the results of the job parsing operation. 
+
+    This list of possible values for **desired_enrichments** and the enrichments the correspond to are as follows:
+    
+    |desired_enrichments | [Enrichment(s)](#Enrichments) |
+    | --- | --- |
+    | `company_norm` | Normalized Company Name |
+    | `geocoding` | Company Geography, Job Geography |
```

Comment:
oh, you might not need what I wrote above then, since you're explaining here that geocoding enrichment entails both company and job locations. I might still rephrase this a bit as "Company location and job location," I think that's a bit clearer and the "Geography" part is already implied